### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mayat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
...
+
+# What is Transifex Native?
+
+A new way to localise software within the code - no files no deployments. A [video](https://www.youtube.com/watch?v=SzEOCKNjRZU) is worth a thousand words!
+But if you are a words person here is the [documentation](https://docs.transifex.com/transifex-native-sdk-overview/introduction).
+
+# Try it out yourself
+
+  
+  - [x] Step 1: Clone the repo of your preference (Python, django, JS, React or React Native)
```
Comment (kbairak) [inline README.md]: Actually, you clone the whole repo and `cd` into the folder of your preference

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+  
+
+  - [x] Step 2: Set things up using this guide: it's literally 3 commands. 
+  <ADD GUIDE here>
```
Comment (kbairak) [inline README.md]: This is going to be slightly different per sample app. I believe you can say something along the lines "follow the instructions within each README"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+
+  If you are not up to it we've built a script to do it for you. Just do <command>
+  
+
+  - [x] Step 3: Push your strings to Transifex with this <command>
```
Comment (kbairak) [inline README.md]: For the first sample app, I named it `sh run.sh`. I'll probably use the same name for the others

File: README.md [hunk 4]
```diff
@@ -1,2 +1,54 @@
...
+
+  Note :We've enable MT so that you don't have to do the translations yourself.
+  
+  
+  - [x] Step 4: Pull your translations.
```
Comment (kbairak) [inline README.md]: I thought the point of transifex native was that you don't have to pull the translations :slightly_smiling_face:

File: README.md (hunk 2 above)
Comment (mayat) [reply README.md]: can we keep the "it's literally 3 commands" or sth like that?

**Review by kbairak — COMMENTED** [review]
My previous comments were wrong at some points: I wasn't thinking of a pre-made shell script to push strings, although it's not a bad idea (although we will have to figure out the issue of the secret token). So I am thinking:

1. clone the repo, `cd` into one of the subfolders
2. set up
   1. either run `sh run.sh`
   2. or follow the instructions (no more than 3 commands)
3. play around:
   1. edit some translations in transifex (link to transifex project, mention that the project is public and that the team has auto_join=true)
   2. tweak the code
4. push new translations (you will need the secret key, contact us)
   1. either run `./sh push <SECRET_TOKEN>`
   2. or follow the instructions (1 command, needs secret token)

About the secret token. I think the READMEs here should mention that people will need to contact us, but we can also include it in the email we send out.

Another note: I am having trouble thinking of sandbox apps for "vanilla" python and javascript. One option could be an interactive cli app, for example:

```sh
$ sh run.sh
Hello, which language would you like to translate this string to?

    Hello world

[1: Greek, 2: French, q: Quit] 2

    Bonjour le monde

Which language would you like to translate this string to?

    Hello world

[1: Greek, 2: French, q: Quit] q

Goodbye
$
```

For vanilla javascript, we can also make an app identical to the react demo, simply without using react (it would be harder but it proves the functionality of the javascript SDK)

Also, Python ships with a [GUI framework](https://docs.python.org/3/library/tkinter.html) in its standard library. It would be a lot of work, but we could make a demo similar to the react demo but in a GUI instead of a webpage.

**Pull request merged**